src/main/kotlin/io/github/zebrateam/ZebraApplication.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.zebrateam
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+
+
+@SpringBootApplication
+class ZebraApplication
+
+fun main(args: Array<String>) {
+    SpringApplication.run(ZebraApplication::class.java, *args)
+}
```

Comment:
Kotlin do not require anything to have body xD Unless it's needed in specific case

It works fine, because under the hood is create class with static methods with name based on filename.